knowledge-base/how-to-install-fiddler-root-certificate-on-windows.md

```diff
@@ -18,18 +18,22 @@ res_type: kb
 
 #### Description
 
-How to install Fiddler root certificate on Windows
+How to install the Fiddler root certificate on Windows manually.
 
->tip Fiddler Everywhere provides automatic option to install the root trust certificate (administrative account still needed). Use [this documentation section]({%slug decrypt-https-traffic%}) as a reference and use the method below only if you need to add the certificate manually.
+>tip Fiddler Everywhere provides an [option to install the trusted root certificate automatically]({%slug #trust-root-certificate%}#trust-root-certificate). Follow the steps described in the [Solution](#solution) section below only if you need to add the certificate manually.
+
+## Prerequisites
+
+You need the __FiddlerRootCertificate.crt__ file. For the instructions to download the certificate, see [Export Root Certificate]({%slug #trust-root-certificate%}#export-root-certificate).
 
 ## Solution
 
 1. Go to your __Desktop__ and double click on __FiddlerRootCertificate.crt__.
 2. Click the __Install Certificate...__ button.
-3. Choose the Store Location and click __Next__.
+3. Select the __Local Machine__ Store Location and click __Next__.
```

Comment:
There are some comments in the user feedback for this document. They say that we do not specify if the users need to select the Local Machine or Current User certificate store. A user has noted that it works only with the Local Machine store. I didn't verify this.